release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2086,6 +2086,15 @@ As a temporary workaround, the image registry should not be configured as privat
 
 [id="ocp-telco-core-4-17-known-issues_{context}"]
 
+* When the `globallyDisableIrqLoadBalancing` field is set to `true` in the `PerformanceProfile` object, isolated CPUs are listed in the `IRQBALANCE_BANNED_CPULIST` variable instead of the `IRQBALANCE_BANNED_CPUS`. However, changing the `globallyDisableIrqLoadBalancing` field from `true` to `false` does not update the `IRQBALANCE_BANNED_CPULIST` variable correctly. As a result, the number of CPUs available for load rebalancing does not increase, as the reserved CPUs remain in the `IRQBALANCE_BANNED_CPULIST` variable. (link:https://issues.redhat.com/browse/OCPBUGS-42323[*OCPBUGS-42323*])
```

Comment:
`IRQBALANCE_BANNED_CPUS` in the first sentence needs to be followed by a noun.

Reply:
Possibly: 

```suggestion
* When the `globallyDisableIrqLoadBalancing` field is set to `true` in the `PerformanceProfile` object, isolated CPUs are listed in the `IRQBALANCE_BANNED_CPULIST` variable instead of the `IRQBALANCE_BANNED_CPUS` variable. However, changing the value of the `globallyDisableIrqLoadBalancing` field from `true` to `false` does not update the `IRQBALANCE_BANNED_CPULIST` variable correctly. As a result, the number of CPUs available for load rebalancing does not increase, as the reserved CPUs remain in the `IRQBALANCE_BANNED_CPULIST` variable. (link:https://issues.redhat.com/browse/OCPBUGS-42323[*OCPBUGS-42323*])
```

Reply:
Updated as per the suggestion.